### ISSUE TITLE
Dealing with collections keyed by ID without an ID in the entity itself

### ISSUE THREAD

**Issue by adjohu** [paularmstrong/normalizr#68]
Is there any way I can deal with data in the following format?

```
{
  1: {
    name: 'test user'
  }
}
```

As far as I can tell, I can only work with objects that themselves contain an id i.e.

```
{
  name: 'test user',
  id: 1
}
```

**Comment by adjohu** [comment]
@gaearon any feedback on this?

**Comment by davegri** [comment]
Was wondering this too

**Comment by adjohu** [comment]
Quick fix in #73.

**Comment by paularmstrong** [comment]
I am happy to get this as an option in normalizr if anyone would like to pick up work on it! (use #73 as a guide)
